### ISSUE TITLE
examples/*.yaml contain non-YAML example comments

### ISSUE THREAD

**Issue by wking** [openshift/installer#146]
For example, we [have][1]:

```
  # (optional) Extra AWS tags to be applied to created resources.
  #
  # Example: `{ "key" = "value", "foo" = "bar" }`
  # extraTags:
```

But that example should use `:` instead of `=` (openshift/release#1209).  I expect this is because the examples were previously generated from Terraform configurations (using `terraform-examples` until d61abd48, coreos/tectonic-installer#3137).  With the coming asset-graph approach replacing (I think) these config files, it may not be worth cleaning this up.  But I thought I'd file an issue to raise awareness until we get around to dropping those examples.

[1]: https://github.com/openshift/installer/blob/4015335421cc2a22d52bdf0d9ae2fca0e7312380/examples/tectonic.aws.yaml#L74-L77

**Comment by wking** [comment]
I've fixed this with the in-flight #236.  If/when that lands, we can close this without waiting for the rest of the next-gen asset graph.

**Comment by crawford** [comment]
We've gotten rid of the example tectonic configs with the move to the new installer.
